runtime/moonbase/src/lib.rs

```diff
@@ -379,6 +387,7 @@ impl pallet_transaction_payment::Config for Runtime {
 	type WeightToFee = ConstantMultiplier<Balance, ConstU128<{ currency::WEIGHT_FEE }>>;
 	type LengthToFee = LengthToFee;
 	type FeeMultiplierUpdate = FastAdjustingFeeUpdate<Runtime>;
+	type WeightInfo = ();
```

Comment:
We want to use weights benchmarked against this runtime here.

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3224/commits/02c52378971b77cf28e77750c2709ce024e819c2 I noticed that Tanssi computed their own weigh info. Do you think it is required here?

Reply:
Read the comment again. Working on a benchmarked solution. 👍

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3224/commits/5044a87cf24e5166ba4a7a8ecbcb40297593b5e7

NOTE: For the benchmark to succeed, I needed to use a workaround that might render the benchmark inaccurate: please share your feedback on this commit.

Reply:
Reverted. Working on a proper solution involving a new cherry-pick on `polkadot-sdk`.

Reply:
Addressed in:
- moonbeam: https://github.com/moonbeam-foundation/moonbeam/pull/3224/commits/ea3b172e3c387a90d4b1df47865e3ec040f12415
- polkadot-sdk: https://github.com/moonbeam-foundation/polkadot-sdk/commit/a1e8632189c14583581445482a2f6002e85385d9

Reply:
We still need to add the benchmarked `WeightInfo`

Reply:
Need to add the weight info.